sbin/build.sh

```diff
@@ -224,9 +230,9 @@ runTheOpenJDKConfigureCommandAndUseThePrebuildConfigParams()
 buildOpenJDK()
 {
   if [[ "$OS_MACHINE" == "s390x" ]] || [[ "$OS_MACHINE" == "armv7l" ]] ; then
```

Comment:
I'm OK with the parameter / logic being passed in, but as Tim says it needs to be buildable from the CL for anyone.  Might be worth just leaving for now.